loopy/target/c/__init__.py

```diff
@@ -1009,7 +1040,9 @@ def emit_blank_line(self):
 
     def emit_comment(self, s):
         from cgen import Comment
-        return Comment(s)
+        # Attaching empty comment wrapping in block needed
+        # otw compiler error due to empty block
+        return Block([Comment(s), Comment(" ")])
```

Comment:
The remaining failing test is a `DocTestFailure` due to this.

Reply:
I fixed it by adjusting the test. Maybe this is something that should only go in the firedrake branch as well.